metaflow/plugins/list_cli.py

```diff
@@ -0,0 +1,77 @@
+from metaflow._vendor import click
+from metaflow import Flow
+from metaflow.exception import MetaflowNotFound
+import json
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group(help="List objects pertaining to your flow.")
+def list():
+    pass
+
+
+def _fetch_runs(flow_name, num_runs):
+    counter = 1
+    try:
+        flow = Flow(flow_name)
+    except MetaflowNotFound:
```

Comment:
We need to set the correct namespace here to ensure that the `Flow` or `Run` can be found.

Reply:
@romain-intel perhaps you can shed some light here as I think I am confused b/w what Savin is saying and what you are saying elsewhere?  

Happy to have a sync conversation as well around this.  Thank you both for your help